SA3/qa_web/forms.py

```diff
@@ -98,3 +98,6 @@ class UserProfile(forms.Form):
 class EditForm(forms.Form):
     content = forms.Textarea()
 
+class DeleteForm(forms.Form):
```

Comment:
I don't believe this is necessary since you never use it in views.py (outside of importing it)